src/main/java/mate/academy/Main.java

```diff
@@ -2,54 +2,92 @@
-        tomorrowMovieSession.setMovie(fastAndFurious);
-        tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
-
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
-
-        MovieSessionService movieSessionService = null;
-        movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
-
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
-        System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+        final MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
+        final CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        final MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
+
+        // Test MovieService
+        Movie movie1 = new Movie();
+        movie1.setTitle("The Matrix");
+        Movie movie2 = new Movie();
+        movie2.setTitle("Inception");
+
+        movieService.add(movie1);
+        movieService.add(movie2);
+        System.out.println("All movies: " + movieService.getAll());
+        System.out.println("Movie with ID 1: " + movieService.get(1L));
+
+        // Test CinemaHallService
+        CinemaHall hall1 = new CinemaHall();
+        hall1.setCapacity(100);
+        hall1.setDescription("IMAX Hall");
+
+        CinemaHall hall2 = new CinemaHall();
+        hall2.setCapacity(50);
+        hall2.setDescription("Standard Hall");
+
+        cinemaHallService.add(hall1);
+        cinemaHallService.add(hall2);
+        System.out.println("All cinema halls: " + cinemaHallService.getAll());
+        System.out.println("Cinema hall with ID 1: " + cinemaHallService.get(1L));
+
+        // Test MovieSessionService
+        MovieSession session1 = new MovieSession();
+        session1.setMovie(movie1);
+        session1.setCinemaHall(hall1);
+        session1.setShowTime(LocalDateTime.of(2024, 12, 8, 14, 0));
+
+        MovieSession session2 = new MovieSession();
+        session2.setMovie(movie2);
+        session2.setCinemaHall(hall2);
+        session2.setShowTime(LocalDateTime.of(2024, 12, 8, 20, 0));
+
+        movieSessionService.add(session1);
+        movieSessionService.add(session2);
+
+        LocalDate testDate = LocalDate.of(2024, 12, 8);
+        System.out.println("Available sessions for 'The Matrix' on " + testDate + ": "
+                + movieSessionService.findAvailableSessions(movie1.getId(), testDate));
+
+        SessionFactory sessionFactory = new Configuration().configure().buildSessionFactory();
```

Comment:
The `SessionFactory` is being created directly in the `main` method. Consider using dependency injection to manage this resource, ensuring it is properly initialized and closed to avoid potential resource leaks.